### PR TITLE
sorry mirza

### PR DESCRIPTION
**Pull request by kheirmirza**
sempai?????

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # psychic-octo-potato
 A repo used for teaching and practice.
+agian
```
Comment (kiesenverseist) [inline README.md]: noice

Comment (kheirmirza) [inline README.md]: ofcourse

Comment (kheirmirza) [inline README.md]: where are you know

Comment (kheirmirza) [inline README.md]: > 

> 

> 

> 

> 

> 

> 

> 

> 

> 

> 

> 

> 

>

**Pull request merged**